charts/haproxy/templates/deployment.yaml

```diff
@@ -65,6 +77,9 @@ spec:
           volumeMounts:
             - name: haproxy-config
               mountPath: /usr/local/etc/haproxy/
+          {{- if .Values.extraVolumeMounts }}
```

Comment:
can you combine extraVolumes and extraVolumeMounts into same list? maybe by adding some extra field used for mounting?